core/unit_test/TestTeamVector.hpp

```diff
@@ -864,6 +864,88 @@ class TestTripleNestedReduce {
 
 #endif
 
+template <typename ExecutionSpace, bool bInclusive, class Reducer>
+void checkScan() {
+  static constexpr int n = 1000000;
```

Comment:
Why `static constexpr`?   Same question below.

Reply:
This is used for the (static) size of the View. The others are there for convenients.